bare/roles/web/tasks/user.yml

```diff
@@ -5,3 +5,9 @@
     shell: /bin/bash
     home: "{{ mastodon_home }}"
 
+- name: Ensure nginx can enter {{ mastodon_home }}
+  ansible.builtin.file:
+    path: "{{ mastodon_home }}"
+    mode: '0701'
+  when:
+    - ansible_os_family == "RedHat"
```

Comment:
this is hardly a test.
if you place web files in a home dir the webserver needs the execute bit on the path in order to read them.
otherwise you simply get an access denied

Reply:
fixed!